googletest/src/gtest-port.cc

```diff
@@ -146,6 +150,19 @@ size_t GetThreadCount() {
   }
 }
 
+#elif GTEST_OS_AIX
+
+size_t GetThreadCount() {
+  struct procentry64 entry;
```

Comment:
AIX has getprocs and getprocs64, depending on whether you're building for a 32-bit or 64-bit target.
How do you know you're building specifically for 64-bit AIX?